### DISCUSSION TITLE
'Back' button links to wrong page

### DISCUSSION THREAD

**Discussion by garrill** [craftcms/.github#69] [Craft Console] [answered]
Hey, found a small bug on the Craft Console website. 

When viewing information on a single plugin the [← Plugins](https://console.craftcms.com/licenses/plugins) link at the top of the page links back to the Craft Console home page, rather than the Organization's page

So it currently links to:
https://console.craftcms.com/licenses/plugins

But if the license belongs to an organization, it would be better to link to:
https://console.craftcms.com/orgs/ {organization} /licenses/plugins

Thanks

**Comment by benjamindavid** [comment, marked as answer]
The issue should be fixed now, thanks for reporting it @garrill
